databrowser/src/domain/cellComponents/components/cells/customDataArrayCell/CustomDataArrayTable.vue

```diff
@@ -0,0 +1,73 @@
+<!--
+SPDX-FileCopyrightText: NOI Techpark <digital@noi.bz.it>
+
+SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<template>
+  <EditListTable :items="items" :hide-tab-link="true">
+    <template #colGroup>
+      <col class="w-32 md:w-80" />
+    </template>
+
+    <template #tableHeader>
+      <TableHeaderCell>Name</TableHeaderCell>
+    </template>
+
+    <template #tableCols="{ item, index }: { item: string, index: number }">
+      <TableCell>
+        <SelectWithOptionsCell
+          v-if="editable"
+          :options="getOptionsForItem(item)"
+          :value="item"
+          :show-empty-value="item == null"
+          :editable="editable"
+          :show-add-new-value="true"
+          :show-value-as-label-fallback="true"
+          @update="updateItem(index, $event.value)"
+        />
+        <!-- Show translated label if not in edit mode -->
+        <span v-else>{{ getLabelForItem(item) }}</span>
+      </TableCell>
+    </template>
+    <template #noItems>No tags have been defined yet</template>
+    <template #addItems>
+      <EditListAddButton :text="'Add new'" @click="addItems([undefined])" />
+    </template>
+  </EditListTable>
+</template>
+
+<script setup lang="ts">
+import { computed, toRefs } from 'vue';
```

Comment:
@a-crea please remove the import for `computed`, it is never used

Reply:
done!